src/neural-network.js

```diff
@@ -60,6 +75,16 @@ export default class NeuralNetwork {
     }
   }
 
+  /**
+   * 
+   * validateTrainingOptions is set true as default
+   *      When true: setting a training option outside expected range will throw an error
+   *      When false: setting a training option outside expected range will log out and continue
+   */
+  get validateTrainingOptions() { return this._isStrict; }
```

Comment:
Return an object? the name suggests there are multiple

Reply:
I don't see how that name suggests an object is returned.  I did change it from being a getter/setter. Originally I was printing out the a log saying what was happening when you set it to false but I thought that was a bit overkill especially once someone understands already how it works.